staging/src/k8s.io/apimachinery/pkg/api/meta/help.go

```diff
@@ -175,6 +175,9 @@ func SetList(list runtime.Object, objects []runtime.Object) error {
 	slice := reflect.MakeSlice(items.Type(), len(objects), len(objects))
 	for i := range objects {
 		dest := slice.Index(i)
+		if dest.Type() == reflect.TypeOf(runtime.RawExtension{}) {
```

Comment:
A test was using [api.List](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L3372) and this PR changes it to [v1.List](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L3900). `api.List.Items` is slice of `runtime.Object`, while `v1.List.Items` is a slice of `runtime.RawExtension`. This change makes SetList to work for `v1.List`, otherwise it's going to try convert an object to runtime.RawExtension and there is no conversion function to do that.